scripts/release/utils.injector.js

```diff
@@ -166,6 +176,7 @@ const generateMethod = (methodIDs, dataClass, mappedSignatures) => {
 
     for (const signature of data.signatures) {
       result.push({ h3: signature.name });
+      signature.flags.isReadonly && result.push({ p: '<code>readonly</code>' });
```

Comment:
methods are expected to be called not get or set. Hence, read-only doesn't apply here, right?

Reply:
Yes, but I put this for supporting if the JSDoc of method has @readonly.

Reply:
yeah but it still doesn't make sense for methods. Shall we remove it?

Reply:
I think we can keep it because it's an option to document and it's feature from JSDoc.

Reply:
I still insist that we should avoid adding unnecessary code.

Reply:
OK let remove it.